### PR TITLE
dx-1015 added info on MIP and references to it

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: _includes/payment-order-other-features.md [hunk 1]
```diff
@@ -1305,6 +1307,7 @@ elements. When in doubt, please follow the [robustness principle](https://en.wik
 [http-api-problems]: https://tools.ietf.org/html/rfc7807
 [image_disabled_payment_menu]: /assets/img/checkout/test-purchase.png
 [image_enabled_payment_menu]: /assets/img/checkout/guest-payment-menu-450x850.png
+[merchant-identified-payer]: /payment-menu/other-features#merchant-identified-payer
```
Comment (asbjornu) [inline _includes/payment-order-other-features.md:1310]: Won't linking directly to `payment-menu` here be a problem in the Checkout section where I assume `payment-order-other-features.md` is also used?

File: _includes/payment-resource.md [hunk 2]
```diff
@@ -285,3 +285,4 @@ for the given operation.
 {% endcase %}
 
 [user-agent]: https://en.wikipedia.org/wiki/User_agent
+[merchant-identified-payer]: /payment-menu/other-features#merchant-identified-payer
```
Comment (asbjornu) [inline _includes/payment-resource.md:288]: I think `payment-menu` in the URL here is going to cause problems in the other sections including `payment-resource.md`.

Comment (payex-dev) [inline _includes/payment-resource.md:288]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5744986)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5744986&bid=18795412)

File: _includes/field-description-payer-reference.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+{%- if documentation_section == "checkout" -%}
+{%- assign documentation_url = documentation_section | prepend: "/" | append: "/other-features#merchant-identified-payer" -%}
+
+{%- else -%}
+{%- assign documentation_url = documentation_section | prepend: "/" | append: "/other-features#merchant-identified-payer" -%}
+{%- endif -%}
+
+{%- capture text -%}
+The reference to the consumer from the merchant system, like mobile number,
+customer number etc. Used in [Merchant Identified Payer]({{ documentation_url }}).
```
Comment (payex-dev) [inline _includes/field-description-payer-reference.md:12]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5744986)

Comment (payex-dev) [inline _includes/field-description-payer-reference.md:12]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5744986)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5744986&bid=18795412)

File: _includes/field-description-payer-reference.md (hunk 3 above)
Comment (payex-dev) [inline _includes/field-description-payer-reference.md:12]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5744986)

Comment (payex-dev) [inline _includes/field-description-payer-reference.md:12]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5744986)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5744986&bid=18795412)

File: _includes/field-description-payer-reference.md (hunk 3 above)
Comment (payex-dev) [inline _includes/field-description-payer-reference.md:12]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5744986)

Comment (payex-dev) [inline _includes/field-description-payer-reference.md:12]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5744986)

Comment (asbjornu) [inline _includes/field-description-payer-reference.md:12]:
```suggestion
customer number etc. Used in [Merchant Identified Payer](/{{ documentation_section }}/other-features#merchant-identified-payer).
```

File: _includes/payment-resource.md [hunk 4]
```diff
@@ -197,7 +197,7 @@ Content-Type: application/json
...
 | └➔&nbsp;`description`    | `string(40)` | {% include field-description-description.md documentation_section=documentation_section %}                                                                                                                                                                                                                                                                 |
-| └➔&nbsp;`payerReference` | `string`     | The reference to the payer (consumer/end-user) from the merchant system, like e-mail address, mobile number, customer number etc.                                                                                                                                                                                                                          |
+| └➔&nbsp;`payerReference` | `string`     | The reference to the payer (consumer/end user) from the merchant system. E.g mobile number, customer number etc.                                                                                                     |
```
Comment (asbjornu) [inline _includes/payment-resource.md:200]:
```suggestion
| └➔&nbsp;`payerReference` | `string`     | The reference to the payer (consumer/end user) from the merchant system. Can be mobile number, e-mail address, account number, a UUID, etc.                                                                                                     |
```

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5744986&bid=18795412)

File: _includes/merchant-identified-payer.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+## Merchant Identified Payer
```
Comment (payex-dev) [inline _includes/merchant-identified-payer.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5744986)

File: _includes/payment-resource.md [hunk 6]
```diff
@@ -197,7 +197,8 @@ Content-Type: application/json
...
 | └➔&nbsp;`description`    | `string(40)` | {% include field-description-description.md documentation_section=documentation_section %}                                                                                                                                                                                                                                                                 |
-| └➔&nbsp;`payerReference` | `string`     | The reference to the payer (consumer/end-user) from the merchant system, like e-mail address, mobile number, customer number etc.                                                                                                                                                                                                                          |
+| └➔&nbsp;`payerReference` | `string`     | The reference to the payer (consumer/end user) from the merchant system. Can be mobile number, e-mail address, account number, a UUID, etc.
```
Comment (payex-dev) [inline _includes/payment-resource.md:200]: Codacy found an issue: [[hard-break-spaces] Use two spaces for hard line breaks](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5744986)

File: _includes/merchant-identified-payer.md (hunk 5 above)
Comment (asbjornu) [reply _includes/merchant-identified-payer.md:1]: Add an extra newline at the end of the file, please, @marinafrydenb.

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5744986&bid=18795412)

File: _includes/payment-resource.md (hunk 6 above)
Comment (payex-dev) [inline _includes/payment-resource.md:200]: Codacy found an issue: [[hard-break-spaces] Use two spaces for hard line breaks](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5744986)

File: _includes/merchant-identified-payer.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+information for later use and then store this information linked to the 
+`payerReference` transmitted with the `Purchase` request.
+
+If you want to use Swedbank Pay Payment Menu in some channels like web and
+build your own in other channels like in-app you should also use this 
+functionality. But when making your own menu you will need to look up the 
+different payment tokens (`paymentToken` or `recurrenceToken`) linked to the 
+specific `payerReference`. You should always look up this information for 
+each payment as the payer can have added new payment information through the a 
+web interface since the last time the app was used.
```
Comment (payex-dev) [inline _includes/merchant-identified-payer.md:17]: Codacy found an issue: [[no-consecutive-blank-lines] Remove 1 line after node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5744986)

**Pull request merged**